test/Interop/Cxx/enum/c-enums-NS_OPTIONS-api-notes-renamed-options.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %empty-directory(%t/cache)
```

Comment:
@hyp I think Daniel found the issue, these two tests were sharing a module cache and empty it at the start of the test. If these tests are run concurrently you get weird flaky failures.